test/fixtures/bn.json

```diff
@@ -0,0 +1,105 @@
+{
+  "valid": [
+    {
+      "dec": "0",
+      "hex": "00"
+    },
+    {
+      "dec": "1",
+      "hex": "01"
+    },
+    {
+      "dec": "-1",
+      "hex": "ff"
```

Comment:
I guess it depends how you want this to be used.
You block out the ease of use as a data interchange format if you use '-'
over a sign bit (twos complement).

Otherwise I have to either encode it myself,  or just send the string
itself.
Using twos complement is the standard for this,  as that is it's entire
purpose.

Also,  you already would be encoding twos complement for use in elliptic?
(DER encoding uses twos complement integers)